packages/dialtone-vue2/recipes/leftbar/general_row/general_row.vue

```diff
@@ -207,6 +220,11 @@ export default {
       default: false,
     },
 
+    hasUnreadMentions: {
+      type: Boolean,
+      default: false,
+    },
```

Comment:
Hmm do we actually need to make this a prop? We can just check `unreadMentionCount !== null` or put that statement into a `hasUnreadsMentions()` computed.

The only reason hasUnreads is it's own prop is because it bolds the text of the row, which needs to be separate from the actual unreadCount.